docs/api-reference/react/deckgl.md

```diff
@@ -23,21 +23,20 @@ const App = (data) => (
 );
 ```
 
-Like any React component, `DeckGL` can accept child components. Child components are often maps (e.g. the `StaticMap` component from react-map-gl), but can be any React components.
+Like any React component, `DeckGL` can accept child components. Child components are often maps (e.g. the `Map` component from react-map-gl), but can be any React components, like deck.gl [Widgets](./overview.md#using-react-wrapped-widgets).
```

Comment:
Special React components include:
- base maps such as ... that can synchronize with deck.gl's view state.
- deck.gl Widgets which can display and control deck.gl state.